manim/utils/qhull.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python
+from __future__ import annotations
+
+import numpy as np
+
+
+class Point:
```

Comment:
There already exists a `Point` mobject in Manim: it's `manim.mobject.types.point_cloud_mobject.Point`. Whether we should actually keep this `Point` in Manim or not is pretty debatable, but, in the meantime, let's change this class' name to something else to avoid name clashes.

Since this is merely an auxiliar class which is only used in this file, maybe it could be prefixed with an underscore, or be called `QuickHullPoint`.
```suggestion
class QuickHullPoint:
```